frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form.tsx

```diff
@@ -0,0 +1,174 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { cephBlockPoolResource } from '../constants/resources';
+import { Dropdown } from '@patternfly/react-core/dist/js/components/Dropdown/Dropdown';
+import { DropdownItem } from '@patternfly/react-core/dist/js/components/Dropdown/DropdownItem';
+import { DropdownToggle } from '@patternfly/react-core/dist/js/components/Dropdown';
+import { CaretDownIcon } from '@patternfly/react-icons';
+
+const PoolResourceComponent: React.FC<PoolParams> = ({ onParamChange }) => {
+  const [data, loaded, loadError] = useK8sWatchResource<K8sResourceKind[]>(cephBlockPoolResource);
+  const [isOpen, setOpen] = React.useState<boolean>(false);
+  const [poolName, setPoolName] = React.useState('');
```

Comment:
```suggestion
  const [poolName, setPoolName] = React.useState<string>('');
```

Reply:
Implicit  from `''`. It would only be necessary when used `null`.

Reply:
Since type has been added to `isOpen` too, either we remove that too or add it here.